corpus_studies/solidity_state_study/slither_plugins/examples/LotteryContract.sol

```diff
@@ -0,0 +1,222 @@
+// SPDX-License-Identifier: UNLICENSED
```

Comment:
Without a license, we can't use this code, unfortunately, unless we get permission from the author.